### PR TITLE
fix: proper attribute value handling (color prop)

### PR DESCRIPTION
**Pull request by mkucmus**
adds "color" property conditionally.

### CODE REVIEW

File: packages/default-theme/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     }
   },
   "dependencies": {
+    "@babel/preset-env": "^7.8.2",
```
Comment (patzick) [inline packages/default-theme/package.json]: why is this needed?

Comment (mkucmus) [reply packages/default-theme/package.json]: It was probably caused obsolete node_modules.
